utils.py

```diff
@@ -124,3 +125,40 @@ def floatable(str):
     else:
         del parser.rargs[:len(value)]
     setattr(parser.values, option.dest, value)
+
+
+class ColorStr(object):
```

Comment:
what motivated the string-like object?

Overall, PR looks good, merging anyhow since tests pass